types.d.ts

```diff
@@ -52,6 +52,8 @@ export interface BlogSettings {
   style?: string;
   /** URL to open graph image. Can be relative. */
   ogImage?: string;
```

Comment:
maybe use `ogImage: string | {url:string, twitterCard:...}` instead of adding new option?

Reply:
@ije 
Thanks you! I'll try to fix it.